nyaa/views/torrents.py

```diff
@@ -119,16 +118,24 @@ def edit_torrent(torrent_id):
         if editor.is_trusted:
             torrent.trusted = form.is_trusted.data
 
-        deleted_changed = torrent.deleted != form.is_deleted.data
         if editor.is_moderator:
+            deleted_changed = torrent.deleted != form.is_deleted.data
             torrent.deleted = form.is_deleted.data
+            locked_changed = torrent.comment_locked != form.is_comment_locked.data
+            torrent.comment_locked = form.is_comment_locked.data
 
         url = flask.url_for('torrents.view', torrent_id=torrent.id)
-        if deleted_changed and editor.is_moderator:
-            log = "Torrent [#{0}]({1}) marked as {2}".format(
-                torrent.id, url, "deleted" if torrent.deleted else "undeleted")
-            adminlog = models.AdminLog(log=log, admin_id=editor.id)
-            db.session.add(adminlog)
+        if editor.is_moderator:
+            if deleted_changed:
+                log = "Torrent [#{0}]({1}) marked as {2}".format(
+                    torrent.id, url, "deleted" if torrent.deleted else "undeleted")
+                adminlog = models.AdminLog(log=log, admin_id=editor.id)
+                db.session.add(adminlog)
+            if locked_changed:
+                log = "Torrent [#{0}]({1}) marked as {2}".format(
+                    torrent.id, url, "locked" if torrent.comment_locked else "unlocked")
```

Comment:
"comments locked" and "comments unlocked" maybe since we're being more specific with the locked status elsewhere.